ansible/roles/satellite-installation/tasks/satellite_installation.yml

```diff
@@ -2,9 +2,13 @@
 - name: Colleting host name
   shell: hostname
   register:  hostname_output
+  tags:
+    - satellite_installation 
 
 - shell: /sbin/ip address show eth0 | grep "\<inet\>" | awk '{print $2}' | awk -F / '{print $1}'
```

Comment:
You can use the fact `ansible_default_ipv4.address` instead.

Consider this example playbook:

```yaml
- hosts: localhost
  gather_facts: yes
  tasks:
    - debug:
        msg: "{{ansible_default_ipv4.address}}"
```

Reply:
Done

Reply:
Changed